opengever/dossier/actions.py

```diff
@@ -40,6 +40,9 @@ def is_move_items_available(self):
     def is_pdf_dossierlisting_available(self):
         return True
 
+    def is_transfer_dossier_responsible_available(self):
+        return True
```

Comment:
```suggestion
        return api.user.has_permission('Modify portal content', obj=self.context)
```

same for the other context actions. But not for the listing-action

Reply:
Did it on the `actions.py ` for class `DossierContextActions`